spec/requests/authentication_pages_spec.rb

```diff
@@ -39,4 +40,41 @@
       end
     end
   end
+
+  describe "authorization" do
+
+    describe "for non-signed-in users" do
+      let(:user) { FactoryGirl.create(:user) }
+
+      describe "in the Users controller" do
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.